python/core/auto_generated/pointcloud/qgspointcloudattribute.sip.in

```diff
@@ -77,6 +79,19 @@ Returns ``True`` if the specified data ``type`` is numeric.
     sipRes = PyUnicode_FromString( str.toUtf8().constData() );
 %End
 
+    static void getPointXYZ( const char *ptr, int i, std::size_t pointRecordSize, int xOffset, QgsPointCloudAttribute::DataType xType,
```

Comment:
Can you hide this method from sip? We shouldn't be exposing the methods which deal with raw char pointers to python, as it has no way to obtain this data in the first place to have anything meaningful to call the method using...

Reply:
Sorry I just forgot to add SIP_SKIP

Reply:
Thanks!